docs/application-information.md

```diff
@@ -6,7 +6,7 @@
 4. [**Application log**](#application-log)
 5. [**Log categories**](#log-categories)
 
-![Application information annotated](/ping-viewer/images/viewer/annotated/application-information-annotated.png)
+![Application information annotated](./images/viewer/annotated/application-information-annotated.png)
```

Comment:
Did you check the generated html with simple http server to check if the generated address will be correct, ping-viewer is hosted under: https://docs.bluerobotics.com/ping-viewer/

Reply:
Yes. You can run `mkdocs build && python3 -m http.server` on the root and see that the ping-viewer page runs correctly.

Reply:
I can be wrong, but our website is managed by github pages where each repository has a prefix, as you can see each image has a ping-viewer prefix because of that (IIRC).
With this PR such prefix will not exist anymore:
![image](https://user-images.githubusercontent.com/1215497/109195450-49edc700-7779-11eb-98c3-a8e7a2e9583d.png)
So https://docs.bluerobotics.com/image... wil fail, since it does not exist, github pages will be waiting for https://docs.bluerobotics.com/ping-viewer/images...
I think that you can accomplish such thing with: site_url
https://www.mkdocs.org/user-guide/configuration/#site_url

Reply:
The whole reason I did this change was because html-proofer failed on my actions and `mkdocs serve` didn't get the images location, so I thought the paths were indeed wrong, but I tested the serving with python's http.server and runnned html-proofer locally, ans both worked flawlesly, so I imagine there's something wrong with `mkdocs serve` and I did something wrong setting up html-proofer on the gh action.

I will revert this change.

PS: from what I saw, `site_url` on MkDocs is just for setting the URL for the header href.

Reply:
In the end, the problem was that I was running html-proofer from inside the ping-viewer folder...

God.

Reply:
You can improve the README to avoid future pain :)

Reply:
Which README?